packages/web-app-files/src/composables/actions/files/useFileActionsDownloadArchive.ts

```diff
@@ -115,6 +115,9 @@ export const useFileActionsDownloadArchive = ({ store }: { store?: Store<any> }
           if (resources.length === 1 && !resources[0].isFolder) {
             return false
           }
+          if (resources.length > 1 && resources.some((r) => isProjectSpaceResource(r))) {
+            return false
+          }
```

Comment:
somehow the isLocationSpacesActive 'generic'  returns true again, this would be a quick fix only :(